package.json

```diff
@@ -70,17 +70,15 @@
     "test:screenshots": "npm run test:screenshots:update -- --verify",
     "test:screenshots:update": "npm run prestart && dockers/Screenshotter/screenshotter.sh",
     "test:perf": "npm run prestart && NODE_ENV=test node test/perf-test.js",
-    "clean": "rm -rf build/* node_modules/",
+    "clean": "rm -rf dist/* node_modules/",
```

Comment:
This is old, but perhaps we should use `rimraf` instead of `rm -rf` (for Windows users, presumably).  Also, I don't see why we delete `dist/*` instead of `dist/` here and in `build`.  It seems slightly better to delete the entire directory instead of leaving it blank.

Reply:
@edemaine As `rimraf` resides in the `node_modules`, it is impossible to delete `node_modules` with `rimraf` 😄

Reply:
I'll change `dist/*` to `dist/`.